selenium/che-selenium-core/src/main/java/org/eclipse/che/selenium/core/workspace/WorkspaceTemplate.java

```diff
@@ -30,7 +30,7 @@
   UBUNTU_GO("ubuntu_go.json"),
   UBUNTU_JDK8("ubuntu_jdk8.json"),
   UBUNTU_LSP("ubuntu_with_c_sharp_lsp.json"),
-  UBUNTU_CAMEL("ubuntu_jdk8_with_camel_ls.json");
```

Comment:
It's not just "Apache Camel"(C) ls, but it is based on `spring-boot` image. More relevant name would sound like the following:
`spring_boot_with_apache_camel_ls`

Reply:
Ok, I'll fix it